web/src/views/repo/settings/Extensions.vue

```diff
@@ -1,17 +1,13 @@
 <template>
   <Settings :title="$t('extensions')" :description="$t('extensions_description')" docs-url="docs/usage/registries">
-    <div class="mb-4 flex flex-row items-center border-b pb-4 dark:border-gray-600">
-      <h1 class="text-color ml-2 text-xl">{{ $t('extensions') }}</h1>
-    </div>
-
-    <div class="flex flex-col">
-      <span class="text-color font-bold">{{ $t('extensions_signatures_public_key') }}</span>
-      <span class="text-color">{{ $t('extensions_signatures_public_key_description') }}</span>
-      <pre class="code-box">{{ signaturePublicKey }}</pre>
-    </div>
-
-    <div class="mt-4 flex flex-col border-t-1 dark:border-gray-600">
+    <div class="mt-4 flex flex-col">
```

Comment:
The div could even be removed as there is only a single form inside (at least what I did recently 🙈).